iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/TransportClientTests.java

```diff
@@ -1,1061 +0,0 @@
-/*
```

Comment:
The TransportClient was once deprecated, and now deleted, so we don't need e2e tests for it anymore